.dockerignore

```diff
@@ -0,0 +1,10 @@
+.git
+.halcyon
+.stack-work
+Dockerfile
+dist
```

Comment:
I don't think we'll see this directory show up unless someone uses bare `cabal`.